### PR TITLE
#223: Updating / Fixing !serverinfo

### PR DESCRIPTION
**Pull request by Azratosh**
# 223: Updating / Fixing `!serverinfo`
## Contents
This PR updates the Discord Nitro boost levels displayed in `!serverinfo` corresponding to Discord's changes to server boost levels. The missing labels for the guild features `ROLE_ICONS` and `NEW_THREAD_PERMISSIONS` are also added.

### Note
The `NEW_THREAD_PERMISSIONS` guild feature flag [does not have any documentation](https://discord.com/developers/docs/resources/guild) yet as of the time of writing this PR. An educated guess has been made as to what this feature does.

## Testing
The changes of both https://github.com/PKlempe/SAM/commit/a9e282b8440f35c90c9642d13e930a307ec408ac and https://github.com/PKlempe/SAM/commit/da8701897e202f0d51b672625336e95d2e53d7eb should be tested *in production,* as test guilds (most likely) do not have any server boosts. Since these changes only affect displayed text, testing in production poses no risk.

### CODE REVIEW

**Comment by PKlempe** [comment]
One can find some information about the new thread permissions [here](https://support.discord.com/hc/de/articles/4403205878423-Threads#h_01FDGC4JW2D665Y230KPKWQZPN).

**Review by PKlempe — CHANGES_REQUESTED** [review]

File: bot/utility/utility.py [hunk 1]
```diff
@@ -316,6 +317,7 @@ def generate_features_list(features: List[str]) -> str:
         "PREVIEW_ENABLED": "Servervorschau",
         "THREADS_ENABLED": "Threads",
         "PRIVATE_THREADS": "Private Threads",
+        "NEW_THREAD_PERMISSIONS": "Rechte für neue Threads",
```
[the remark this answers is not in the document]
Comment (Azratosh) [reply bot/utility/utility.py:320]: Sounds much better, thanks!

File: bot/utility/utility.py [hunk 2]
```diff
@@ -316,6 +317,7 @@ def generate_features_list(features: List[str]) -> str:
         "PREVIEW_ENABLED": "Servervorschau",
         "THREADS_ENABLED": "Threads",
         "PRIVATE_THREADS": "Private Threads",
+        "NEW_THREAD_PERMISSIONS": "Neue Thread-Berechtigungen"
```
Comment (PKlempe) [inline bot/utility/utility.py:320]:
```suggestion
        "NEW_THREAD_PERMISSIONS": "Neue Thread-Berechtigungen",
```

Comment (PKlempe) [reply bot/utility/utility.py:320]: I forgot the comma. 🥲

**Review by PKlempe — CHANGES_REQUESTED** [review]

**Review by PKlempe — APPROVED** [review]

**Pull request merged**